benchmark/dns/lookup.js

```diff
@@ -5,7 +5,7 @@ const lookup = require('dns').lookup;
 
 const bench = common.createBenchmark(main, {
   name: ['', '127.0.0.1', '::1'],
-  all: [true, false],
+  all: ['true', 'false'],
```

Comment:
Another possibility is to leave them as strings but change the assignment to something like:  `const all = conf.all === 'true' ? true : false;`

To me, this has the advantage of being more readable and also seems to be at least a little bit in line with [what some other benchmarks do](https://github.com/nodejs/node/blob/master/benchmark/dgram/bind-params.js#L16-L17). Of course, @BridgeAR's suggestion works too if that seems preferable.